backend/routes/form.py

```diff
@@ -150,3 +162,18 @@ def add(req: Request,
     verify_admin(req.state.authenticated, session)
     form = crud.add_form(session=session, name=name)
     return form.serialize
+
+
+@form_route.get("/webform_editor/{id:path}",
```

Comment:
@wayangalihpratama can we just have one endpoint with query params `?edit=true`?

Reply:
sure, I will make an update for this mas @dedenbangkit